assets/files/plugins.json

```diff
@@ -0,0 +1,42 @@
+[
```

Comment:
so json is really awkward to edit
make this a normal JS file

Reply:
Switched to normal JS variable defined locally